latest_migrations.manifest

```diff
@@ -5,6 +5,7 @@ contenttypes: 0002_remove_content_type_name
 ee: 0015_add_verified_properties
 otp_static: 0002_throttling
 otp_totp: 0002_auto_20190420_0723
+posthog: 0374_scheduled_change
```

Comment:
I don't think this belongs here?

Reply:
Hmm strange, this line was added in my previous PR, but then was removed by ... your PR 😅 

So let's add it back :)

Reply:
There should only be 1 posthog migration here and it should be the most recent one which is 0375

Reply:
You're not even migrating anything in this PR, I'm confused? 😆

Reply:
There should only be one entry for `posthog:` that corresponds with the latest migration by number in `posthog/migrations`!

Reply:
oh I see, thanks for clarifying!